app/uk/gov/hmrc/cgtpropertydisposalsfrontend/connectors/CGTPropertyDisposalsConnector.scala

```diff
@@ -30,33 +30,42 @@ import scala.concurrent.{ExecutionContext, Future}
 @ImplementedBy(classOf[CGTPropertyDisposalsConnectorImpl])
 trait CGTPropertyDisposalsConnector {
 
-  def getBusinessPartnerRecord(nino: NINO)(implicit hc: HeaderCarrier): EitherT[Future, Error, HttpResponse]
+  def getBusinessPartnerRecord(nino: NINO, name: Name, dob: DateOfBirth)(
+    implicit hc: HeaderCarrier
+  ): EitherT[Future, Error, HttpResponse]
 
-  def subscribe(subscriptionDetails: SubscriptionDetails)(implicit hc: HeaderCarrier): EitherT[Future, Error, HttpResponse]
+  def subscribe(subscriptionDetails: SubscriptionDetails)(
+    implicit hc: HeaderCarrier
+  ): EitherT[Future, Error, HttpResponse]
 }
 
 @Singleton
-class CGTPropertyDisposalsConnectorImpl @Inject() (
-    http: HttpClient,
-    servicesConfig: ServicesConfig
-)(implicit ec: ExecutionContext) extends CGTPropertyDisposalsConnector {
+class CGTPropertyDisposalsConnectorImpl @Inject()(http: HttpClient, servicesConfig: ServicesConfig)(
+  implicit ec: ExecutionContext
+) extends CGTPropertyDisposalsConnector {
 
   val baseUrl: String = servicesConfig.baseUrl("cgt-property-disposals") + "/cgt-property-disposals"
 
   def bprUrl(nino: NINO): String = s"$baseUrl/${nino.value}/business-partner-record"
 
   val subscribeUrl: String = s"$baseUrl/subscribe"
 
-  def getBusinessPartnerRecord(nino: NINO)(implicit hc: HeaderCarrier): EitherT[Future, Error, HttpResponse] =
+  def getBusinessPartnerRecord(nino: NINO, name: Name, dob: DateOfBirth)(
+    implicit hc: HeaderCarrier
+  ): EitherT[Future, Error, HttpResponse] =
     EitherT[Future, Error, HttpResponse](
-      http.get(bprUrl(nino))
+      http
+        .get(bprUrl(nino))
```

Comment:
isn't this a POST with a body now?

Reply:
Yes! Good spot. The backend endpoint changed late yesterday.